sdk/build.gradle

```diff
@@ -63,16 +63,16 @@ dependencies {
     implementation "com.squareup.okhttp3:okhttp:$okHttpVersion"
     implementation "com.squareup.okhttp3:okhttp-urlconnection:$okHttpVersion"
     api "net.danlew:android.joda:2.10.0"
-    implementation "com.squareup.retrofit2:retrofit:2.3.0"
-    implementation "com.squareup.retrofit2:converter-gson:2.3.0"
+    implementation "com.squareup.retrofit2:retrofit:2.4.0"
+    implementation "com.squareup.retrofit2:converter-gson:2.4.0"
     implementation "com.google.code.gson:gson:2.8.5"
     implementation "javax.inject:javax.inject:1"
     implementation "javax.annotation:javax.annotation-api:1.3.2"
 
     testImplementation "com.atlassian.oai:swagger-request-validator-core:1.4.2"
```

Comment:
https://bitbucket.org/atlassian/swagger-request-validator/src/master 2.0.2 has API changes but it also raised some other violation error (can't remember what it was).